fastlane_bot/events/exchanges/base.py

```diff
@@ -124,13 +123,6 @@ async def get_fee(address: str, contract: AsyncContract) -> float:
         """
         pass
 
-    @abstractmethod
```

Comment:
This function needs to be renamed & modified instead of removed because the `get_pool` function that it's now using is something completely different.

Reply:
I don't understand, the function `get_pool_with_multicall` was renamed to `get_pool` in every class in which it was actually useful, and removed in all other classes. Why does it need to be renamed here instead of removed?